notebooks/create_alerts.py

```
@@ -0,0 +1,47 @@
+# COMMAND ----------
+alert = w.alerts.create(
+    alert=sql.CreateAlertRequestAlert(
+        condition=sql.AlertCondition(
+            operand=sql.AlertConditionOperand(column=sql.AlertOperandColumn(name="prediction_drift")),
+            op=sql.AlertOperator.EQUAL,
+            threshold=sql.AlertConditionThreshold(value=sql.AlertOperandValue(double_value=1)),
+        ),
+        display_name=f"hotel-reservations-alert-query-{time.time_ns()}",
+        query_id=query.id,
+    )
+)
```

Ruff (0.7.0)

No newline at end of file

Add trailing newline

(W292)